src/plugins/usage_collection/server/routes/stats/stats.ts

```diff
@@ -55,9 +60,9 @@ export function registerStatsRoute({
 }) {
   const getUsage = async (
     esClient: ElasticsearchClient,
-    savedObjectsClient: SavedObjectsClientContract | ISavedObjectsRepository,
+    savedObjectsClient: SavedObjectsClientContract,
```

Comment:
I don't know why TS didn't complain earlier. We migrated to `SavedObjectsClientContract`-only way back in #93289 🤷